src/_igraph/graphobject.c

```diff
@@ -12605,6 +12666,26 @@ struct PyMethodDef igraphmodule_Graph_methods[] = {
     "@param repeats: the number of repeats\n"
   },
 
+  {"Realize_Degree_Sequence", (PyCFunction) igraphmodule_Graph_Realize_Degree_Sequence,
+    METH_VARARGS | METH_CLASS | METH_KEYWORDS,
+    "Realize_Degree_Sequence(outdeg, indeg, allowed_edge_types, method)\n--\n\n"
+    "Generates a graph from a degree sequence.\n\n"
+    "@param outdeg: the degree sequence of an undirected graph (if indeg=None), "
+    "or the out-degree sequence of a directed graph.\n"
+    "@param indeg: None to generate an undirected graph, the in-degree sequence "
+    "to generate a directed graph.\n"
+    "@param allowed_edge_types: for directed graphs, only 'simple_sw' is currently \n"
```

Comment:
I'd rather not provide details here; if we update the C core and add other implementations, there's a 99% chance that we forget to update the documentation here. I'll remove the details in a separate commit and just mention that we support whatever the C core supports.